skimage/morphology/convex_hull.py

```diff
@@ -86,8 +112,8 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
         gridcoords = np.reshape(np.mgrid[tuple(map(slice, image.shape))],
                                 (ndim, -1))
         # A point is in the hull if it satisfies all of the hull's inequalities
```

Comment:
This comment should probably be moved to `_test_ineq`, above.

Reply:
Done